src/test/java/lesser/citibike/CitiBikeMapServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package lesser.citibike;
+
+import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import io.reactivex.rxjava3.core.Single;
+import lesser.citibike.map.*;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+public class CitiBikeMapServiceTest {
+
+    @Test
+    public void getRouteSuccess() {
+        // given
+        final CitiBikeMapService serviceMock = mock(CitiBikeMapService.class);
```

Comment:
If you are testing CitiBikeMapService then you shouldn't be using a mock for it.